components/ca-barcode-scanner.js

```diff
@@ -0,0 +1,116 @@
+/* global cordova */
+define(['./helpers/create-element.js', 'document-register-element'], (createElement) => {
+    const txtProp = ('textContent' in document.createElement('i')) ? 'textContent' : 'innerText';
```

Comment:
Can be removed and just use textContent. innerText is not an equivalent and we don't care about IE8.